### PR TITLE
General UI improvements

### LINKED ISSUE #225: Bugfix latest date sorting
-When sorting by latest in the forms tab the sorting is not behaving as expected, e.g. 4th October above 17th October

### LINKED ISSUE #250: Created and modified fields for replies/forms
As a user I'd like to apply same logic that's used for adding status field for replies/forms to also add created and modified fields.

### LINKED ISSUE #254: Add texts to user management screens dialog tooltips
As a user I'd like to have actual texts in user management screen dialogs tooltips.

### LINKED ISSUE #255: Audit log entry actor
As a user I'd like to see the actual actors of audit log entries. At the time it wasn't possibly to implement due to API restrictions but now it is.

### PR DESCRIPTION
**Pull request by villejuutila**
Resolves #255
Resolves #254
Resolves #250
Resolves #225

Also small other things here and there.

- Manual link is disabled if URL of manual is not present
- Better handling of requesting user information from API to get forms last modifiers
- Refined the handling of adding/removing forms metadata fields, e.g. status, created, modified

### CODE REVIEW

**Review by EVP1989 — APPROVED** [review]
Approved

File: src/components/screens/public/public-form-screen.tsx [hunk 1]
```diff
@@ -1,10 +1,666 @@
...
+ * Component props
  */
-const PublicFormScreen: React.FC = () => (
-  <div/>
-);
+interface Props {
+}
+
+/**
+ * Component for exhibitions screen
```
Comment (EVP1989) [inline src/components/screens/public/public-form-screen.tsx:35]: Check tsdocs

File: src/components/screens/public/public-form-screen.tsx [hunk 2]
```diff
@@ -1,10 +1,666 @@
...
+ */
+const PublicFormScreen: React.FC<Props> = () => {
+  const AUTOSAVE_COOLDOWN = 500;
+
+  const errorContext = React.useContext(ErrorContext);
+
+  const [ , setLoading ] = useState(false);
+  const [ savingReply, setSaving ] = useState(false);
+  const [ , setSnackbarMessage ] = useState<SnackbarMessage>();
+  const [ , setReplyConfirmVisible ] = useState(false);
```
Comment (EVP1989) [inline src/components/screens/public/public-form-screen.tsx:45]: Check file for unused/unnecessary states

**Review by EVP1989 — COMMENTED** [review]
Added comments

File: src/components/screens/public/public-form-screen.tsx [hunk 3]
```diff
@@ -1,10 +1,666 @@
...
 
 /**
- * Public forms screen component
+ * Component props
  */
-const PublicFormScreen: React.FC = () => (
-  <div/>
-);
+interface Props {
+}
```
[the remark this answers is not in the document]
Comment (villejuutila) [reply src/components/screens/public/public-form-screen.tsx:32]: Resolved

File: src/components/screens/public/public-form-screen.tsx [hunk 4]
```diff
@@ -1,10 +1,666 @@
...
+        onValidationErrorsChange={ onValidationErrorsChange }
+        saving={ savingReply }
+      />
+    );
+  };
+
+  return (
+    /**
+     * Implement layout later
+     */
```
[the remark this answers is not in the document]
Comment (villejuutila) [reply src/components/screens/public/public-form-screen.tsx:608]: Resolved

File: src/components/screens/public/public-form-screen.tsx [hunk 5]
```diff
@@ -1,10 +1,666 @@
...
+  };
+
+  /**
+   * Implement later
+   */
+  const updateReply = async (currentMetaform: Metaform, currentReply: Reply, currentOwnerKey: string | null | undefined) => {
+    const { repliesApi } = apiClient;
+
+    await repliesApi.updateReply({
+      metaformId: metaformId!!,
```
[the remark this answers is not in the document]
Comment (villejuutila) [reply src/components/screens/public/public-form-screen.tsx:285]: Resolved

File: src/components/screens/public/public-form-screen.tsx (hunk 1 above)
Comment (villejuutila) [reply src/components/screens/public/public-form-screen.tsx:35]: Resolved

**Review by EVP1989 — APPROVED** [review]
Approved

**Pull request merged**